resources/js/Pages/Home.vue

```diff
@@ -31,6 +39,13 @@
             width: 300px;
         }
     }
+    .message{
+        position: absolute;
+        width: 40vw;
+        top: 50px;
+        left: 33%;
+        // transform: translate(-50%, -50%);
```

Comment:
```suggestion
        width: 40vw;
        top: 10vh;
        left: 33%;
        z-index: 120;
```
The `z-index` is there to be on top of the menu. and the `10vh` is just here to also support mobile.